public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -458,12 +458,31 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
         <div className={styles.wrapper}>
           <div className={styles.verticalSplitPanesWrapper}>
             <SplitPaneWrapper
-              leftPaneComponents={this.renderPanelAndEditor(styles)}
-              rightPaneComponents={this.renderOptionsPane()}
-              uiState={uiState}
-              updateUiState={updatePanelEditorUIState}
-              rightPaneVisible={uiState.isPanelOptionsVisible}
-            />
+              splitOrientation="vertical"
+              maxSize={-300}
+              paneSize={uiState.rightPaneSize}
+              primary="second"
+              onDragFinished={(size) => {
+                if (size) {
+                  updatePanelEditorUIState({ rightPaneSize: size / window.innerWidth });
+                }
+              }}
+            >
+              <SplitPaneWrapper
+                splitOrientation="horizontal"
+                maxSize={-200}
```

Comment:
-200 ?

Reply:
Keeping the params as they were https://github.com/grafana/grafana/blob/main/public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx#L83

Reply:
ok :)